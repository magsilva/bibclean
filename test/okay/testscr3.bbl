\begin{thebibliography}{1}

\bibitem{CommENT}


\bibitem{foo}


\bibitem{bar}


\end{thebibliography}
