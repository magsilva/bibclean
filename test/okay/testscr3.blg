Warning--entry type for "CommENT" isn't style-file defined
--line 3 of file testscr3.bib
I was expecting a `{' or a `('---line 6 of file testscr3.bib
 : @end 
 :      <cOMMent>
I'm skipping whatever remains of this entry
Repeated entry---line 8 of file testscr3.bib
 : @begin {CommENT
 :                }
I'm skipping whatever remains of this entry
Warning--entry type for "foo" isn't style-file defined
--line 11 of file testscr3.bib
Warning--entry type for "bar" isn't style-file defined
--line 12 of file testscr3.bib
Repeated entry---line 13 of file testscr3.bib
 : @end{bar
 :         }
I'm skipping whatever remains of this entry
Repeated entry---line 14 of file testscr3.bib
 : @end{foo
 :         }
I'm skipping whatever remains of this entry
I was expecting a `{' or a `('---line 15 of file testscr3.bib
 : @end 
 :      <cOMMent>
I'm skipping whatever remains of this entry
Repeated entry---line 17 of file testscr3.bib
 : @begin {CommENT
 :                }
I'm skipping whatever remains of this entry
I was expecting a `{' or a `('---line 20 of file testscr3.bib
 : @end 
 :      <cOMMent>
I'm skipping whatever remains of this entry
Repeated entry---line 22 of file testscr3.bib
 : @begin {CommENT
 :                }
I'm skipping whatever remains of this entry
I was expecting a `{' or a `('---line 25 of file testscr3.bib
 : @end 
 :      <cOMMent>
I'm skipping whatever remains of this entry
Warning--to sort, need author or key in CommENT
Warning--to sort, need author or key in foo
Warning--to sort, need author or key in bar
(There were 9 error messages)
