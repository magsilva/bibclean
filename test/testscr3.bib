@Comment "Old style comment"

@Begin {CommENT}
This is a block comment
with two lines.
@End <cOMMent>

@Begin {CommENT}
This is a block comment
with nested environments.
@Begin{foo}
@Begin{bar}
@End{bar}
@End{foo}
@End <cOMMent>

@Begin {CommENT}
This is a block comment with two lines and embedded braces: {one}
{two} {three}
@End <cOMMent>

@Begin {CommENT}
This is a block comment with two lines and embedded braces: {one}
{two} {three}.  Here are some mismatched braces: { { { } } } } }
@End <cOMMent>

