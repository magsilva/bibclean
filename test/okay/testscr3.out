@Comment{Old style comment}

@Comment{
This is a block comment
with two lines.
}

@Comment{
This is a block comment
with nested environments.
@Begin{foo}
@Begin{bar}
@End"bar"
@End"foo"
}

@Comment{
This is a block comment with two lines and embedded braces: {one}
{two} {three}
}

@Comment{
This is a block comment with two lines and embedded braces: {one}
{two} {three}.  Here are some mismatched braces: { { { } } } {} {}
}
